Formula/m/moreutils.rb

```diff
@@ -1,9 +1,8 @@
 class Moreutils < Formula
   desc "Collection of tools that nobody wrote when UNIX was young"
   homepage "https://joeyh.name/code/moreutils/"
-  url "https://git.joeyh.name/git/moreutils.git",
-      tag:      "0.67",
-      revision: "ee8e013cd5398c5fb34fb19a24db0f3b6185bac3"
```

Comment:
not quite sure why we used git checkout before, but this is a good improvement

- relates to https://github.com/Homebrew/homebrew-core/pull/12651